application/app.py

```diff
@@ -16,15 +15,8 @@
 ###############################################
 
 import os
-import ConfigParser
-from PyQt4.QtGui import *
+from PyQt5.QtWidgets import *
```

Comment:
Do we need `import *`? Do we use all library parts, or should we instead pick the ones we need?